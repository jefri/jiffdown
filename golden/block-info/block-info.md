> {aside.info} Information in an `aside`.
