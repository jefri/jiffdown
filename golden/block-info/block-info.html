<aside class="info">
<p>Information in an <code>aside</code>.</p>
</aside>
